src/StoreApi/Schemas/V1/CrossSellsItemSchema.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Schemas\V1;
+
+/**
+ * CrossSellsItemSchema class.
+ */
+class CrossSellsItemSchema extends ProductSchema {
+
+	/**
+	 * The schema item name.
+	 *
+	 * @var string
+	 */
+	protected $title = 'cart_cross_sells_item';
+
+	/**
+	 * The schema item identifier.
+	 *
+	 * @var string
+	 */
+	const IDENTIFIER = 'cart-cross-sells-item';
+
+	/**
+	 * Cross-sells schema properties.
+	 *
+	 * @return array
+	 */
+	public function get_properties() {
+		$properties = parent::get_properties();
+
+		return [
+			'id'                => $properties['id'],
+			'name'              => $properties['name'],
+			'permalink'         => $properties['permalink'],
+			'prices'            => $properties['prices'],
+			'images'            => $properties['images'],
+			'average_rating'    => $properties['average_rating'],
+			self::EXTENDING_KEY => $this->get_extended_schema( self::IDENTIFIER ),
+		];
```

Comment:
I noticed that we need one more property: `on_sale`. This property controls the display of the sales badge:

<img width="534" alt="Screenshot 2022-06-30 at 11 25 09" src="https://user-images.githubusercontent.com/3323310/176642396-f92ca0b6-b8b4-4bcc-a03c-393e3fef8e71.png">

Reply:
I stopped extending ProductSchema, so [now Cross-Sell items will use it directly](https://github.com/woocommerce/woocommerce-blocks/pull/6635/commits/d27a1e4c2c87ade2190a939722c5b4a9805f91ce). Everything from Product is available 🙂.